_posts/2023-04-05-cockpit-289.md

```diff
@@ -0,0 +1,123 @@
+
+![Screenshot from 2023-04-05 12-41-47]({{ site.baseurl }}/images/289-accounts-improve-password-setting.png)
+
+## Machines: Show "pending changes" when configuration changes require a shutdown to take effect
+
+State: **OPEN** (will not be included) @ <https://github.com/cockpit-project/cockpit-machines/pull/998>
+{:.warn}
+
+Add label which indicated whetever VM has pending changes and needs shutdown, and shows what specific configuration changes cause it.
+
+## Machines: Show an alert when virtualization is disabled in BIOS/EFI
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-machines/pull/980>
+{:.note}
+
+![screenshot of if virtualization is disabled in bios/efi we now show message at initial load]({{ site.baseurl }}/images/289-machines-if-virtualization-is-disabled-in-bios-efi-we-now-show-message-at-initial-load.png)
+
+Thanks to Subho Ghosh for this contribution.
+
+## Machines: VSOCK interface
+
+State: **OPEN** (will not be included) @ <https://github.com/cockpit-project/cockpit-machines/pull/969>
+{:.warn}
+
+
+## If storage file fails to get deleted, the disk will still get detached from the VM, but we show a warning:
+![Screenshot from 2022-11-14 18-48-54]({{ site.baseurl }}/images/289-if-storage-file-fails-to-get-deleted-the-disk-will-still-get-detached-from-the-vm-but-we-show-a-warning-.png)
+
+## Podman: Support for pruning unused containers
+
+State: **OPEN** (will not be included) @ <https://github.com/cockpit-project/cockpit-podman/pull/1235>
+{:.warn}
+
+<fill in>
+
+![screenshot of ## prune unused containers support]({{ site.baseurl }}/images/289-prune-unused-containers-support.png)
+
+## Podman: Custom healthcheck actions
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/1233>
+{:.note}
+
+When a container becomes unhealthy a custom action can be configured in Podman, such as restarting the container. This requires Podman 4.3 or higher.
+
+![screenshot of ## specify a custom healthcheck action]({{ site.baseurl }}/images/289-specify-a-custom-healthcheck-action.png)
+
+## Podman: Container list can be sorted
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/1232>
+{:.note}
+
+The podman container section now allows a user to sort on name, owner, CPU, memory and state.
+
+![screenshot of ## sort containers]({{ site.baseurl }}/images/289-sort-containers.png)
+
+## Podman: Explain the difference between user and system sessions
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/1227>
+{:.note}
+
+When creating a container with the system session offer a popover with the differences between the owner types.
+
+Closes: #1067
+
+---
+
+![screenshot of explain the difference between a user and system session]({{ site.baseurl }}/images/289-podman-explain-the-difference-between-a-user-and-system-session.png)
+
+## Development: Cockpit now supports the esbuild bundler
```

Comment:
This can be removed as it's from previous release. My fault not removing the label sorry.